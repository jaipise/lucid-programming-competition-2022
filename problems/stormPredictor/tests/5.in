Storm clouds: true
South winds: true
Air pressure change: true
Silent animals: false