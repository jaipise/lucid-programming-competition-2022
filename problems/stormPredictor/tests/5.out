Wait for the storm to pass.
